src/hooks/use-snapshots.tsx

```diff
@@ -280,10 +286,12 @@ export const SnapshotProvider: React.FC< { children: ReactNode } > = ( { childre
 		}
 		setIsLoading( true );
 		try {
-			const response: { site_count: number; site_limit: number } = await client.req.get( {
-				path: '/jurassic-ninja/usage',
-				apiNamespace: 'wpcom/v2',
-			} );
+			const response: { site_count: number; site_limit: number; site_creation_blocked: boolean } =
+				await client.req.get( {
+					path: '/jurassic-ninja/usage',
+					apiNamespace: 'wpcom/v2',
+				} );
+			setSnapshotCreationBlocked( response.site_creation_blocked );
```

Comment:
What do you think about moving `setActiveSnapshotCount` and  `setSnapshotQuota` from `fetchStats` to here ?

Reply:
Actually initially I have done the reverse and added `setSnapshotCreationBlocked` to `fetchStats` but when refactoring for polling the endpoint, I think this made more sense. 

Generally, I don't have a strong preference but one thing I am worried about is that these changes might have been intentional (meaning for fetching stats), and I would be worried about breaking something that does not necessarily directly relate to the fix in this PR. I would prefer to leave this as is and not touch it.

Reply:
Makes sense, initially I was also thinking about moving `setSnapshotCreationBlocked` to `fetchStats`, but fetchStats is not exported and it would require a bit of refactor. It's ok to let it as it is.